pyproject.toml

```diff
@@ -8,7 +8,154 @@ requires = ["setuptools",
             "extension-helpers"]
```

Comment:
The indentation in this file is all horribly inconsistent and some of it is my fault. Someone needs to crack it in an editor and pick an indentation pattern.

Reply:
I reformatted it on this branch. Please double check.